src/main/java/eu/dissco/core/translator/service/BioCaseService.java

```diff
@@ -370,39 +348,31 @@ private JsonNode cleanupRedundantFields(ObjectNode unitData) {
     return data;
   }
 
-  private void processDigitalMediaObjects(String physicalSpecimenId, Unit unit, String organisationId)
-      throws JsonProcessingException {
+  private List<DigitalMediaObjectEvent> processDigitalMediaObjects(String physicalSpecimenId,
```

Comment:
hmm I think this one is a bit different. The schema object is DigitelEntity and the code one is DigitalMediaObject. So there is not really a conflict here. MultiMediaObjects is only used because this is what it is called in ABCD(EFG)